controllers/agentaction_controller.go

```diff
@@ -713,14 +711,27 @@ func (r *AgentActionReconciler) getAgentVolumes(action *porterv1.AgentAction, pv
 		},
 		)
 	}
-
-	for _, volume := range action.Spec.Volumes {
-		volumes = append(volumes, volume)
+	if !action.IsAgentConfig() {
```

Comment:
Can you add a comment here? It's not immediately clear what's going on and an explanation would help a lot. I can't really tell from the code when we mount this volume.